lib/api.ts

```diff
@@ -300,3 +300,12 @@ const getLastWeekHighlights = (calendarData: Highlights[]) => {
 
   return highlights;
 };
+
+export const getContributorRank = async (name: string) => {
```

Comment:
I'd say add a prop in the `weekSummary` itself specifying the `rank` as a field instead of a new function

Reply:
I try that, sure I will do it again